install/installer/cluster_dev.py

```diff
@@ -110,26 +118,55 @@ def validate(self, document=None, interactive=True):
             )  # Move cursor to end
 
 
-def ask_user(question_name, non_interactive_value=None):
+@typechecked
+class AWSUserNameValidator(Validator):  # pylint: disable=too-few-public-methods
+    """Validate user input"""
+
+    def validate(self, document=None, interactive=True):
+        """
+        Validate user input string to AWS username restrictions
+
+        Raises:
+            ValidationError: If intput string not match regex.
+                Show error message for user and get him change fix error
+        """
+        username = self
+        if interactive:
+            username = document.text
+
+        error_message = 'Please enter a valid AWS username. ' + \
+            'It should have 1-64 only latin letters, ' + \
+            'numbers and symbols: _+=,.@-'
```

Comment:
First of all, you don't need to concatenate subsequent string literals in runtime, and second — it's very easy to make mistakes when EOL escaping is used:
```suggestion
        error_message = (
            'Please, enter a valid AWS username. '
            'It should consist of 1-64 of latin-only letters, '
            'numbers and symbols: _+=,.@-'
        )
```